ui/components/multichain/network-list-menu/network-list-menu.tsx

```diff
@@ -511,7 +515,7 @@ export const NetworkListMenu = ({ onClose }: { onClose: () => void }) => {
       <NetworkListItem
         key={network.chainId}
         chainId={network.chainId}
-        name={network.name}
+        name={name}
```

Comment:
Nit: But I think that `network.name` was a bit better, so we could just revert this change and use what was initially used, WDYT?